packages/terra-brand-footer/src/terra-dev-site/doc/example/BrandFooterExampleCommon.scss

```diff
@@ -0,0 +1,30 @@
+:local {
+  .content-left-anchor {
+    background-color: transparent;
+    color: #fff;
+    text-decoration: none;
+  }
+
+  .content-left-graphics {
```

Comment:
Technically you can add more than one class name to an element. So this could be change to be something like:
```
:local {
  .anchor {
    background-color: transparent;
    color: #fff;
    text-decoration: none;
  }

  .graphics {
    border: 0;
    height: 40px; // setting this explicity for IE11
    width: 40px; // setting this explicity for IE11
  }

  .left-graphics {
    vertical-align: middle;
  }

  .right-graphics {
    vertical-align: top;
  }

  .text {
    letter-spacing: 1px;
  }

  .weighted-text {
    font-weight: 100;
  }
}
```

Then, add multiple where needed, like:

```diff
- <span className={cx('content-left-terratext')}>Terra </span>
+ <span className={cx('text')}>Terra </span>

- <span className={cx('content-left-uitext')}>UI</span>
+ <span className={cx(['text', 'weighted-text'])}>UI</span>
```

Reply:
I implemented `content-left-text` and `weighted-text` as per your suggestion.
But did not touch "left and right graphics" since the `width`and `vertical-align` held different values.